tools/azure_iot_operations_stub_services/src/schema_registry.rs

```diff
@@ -19,15 +20,204 @@ pub const SERVICE_NAME: &str = "schema_registry";
 pub const CLIENT_ID: &str = "schema_registry_service_stub";
 const NAMESPACE: &str = "aio-sr-ns-stub";
 
+// ~~~~~~~~~~~~~~~~~~~DTDL Equivalent Error~~~~~~~
+
+/// Error codes for schema operations.
+#[derive(Debug, Clone)]
```

Comment:
since the stub services don't expose a public API, do we still need to wrap the codegen output? (I'm not opposed, but I'm not sure we need the same safety net here since we only have breaking changes if the end behavior changes)